Vic2ToHoI4/Source/HOI4World/Events/Events.cpp

```diff
@@ -0,0 +1,1345 @@
+#include "Events.h"
```

Comment:
I'm not getting into the details of the events you create, but for readability it may be easier to replace tabs with \t - if that doesn't affect the output. Personal taste, not serious.